src/components/PathList.js

```diff
@@ -0,0 +1,4 @@
+export const ROOT_API = "http://127.0.0.1:8000";
+export const SIGNUP_API = "http://127.0.0.1:8000/api/signup/";
+export const TOKEN_API = "http://127.0.0.1:8000/api/token/";
+
```

Comment:
```suggestion
export const API_DOMAIN = "127.0.0.1:8000";
```

Reply:
Let's do smth like that. In the future, we'll have different domains for different stages